### ISSUE TITLE
test with val.py

### ISSUE THREAD

**Issue by Jillian2017** [fab-jul/imgcomp-cvpr#1]
Hi, I cannot fully understand your parameters in val.py, such as job_ids, what kind of parameter should i give? I very appricate that you can give a detailed explation for others parameters. Thanks a lot.

**Comment by fab-jul** [comment]
Please follow the README. You need to download the checkpoints and extract them to `ckpts`. Then, the possible values for MODEL_ID are given in the README. Let me know if this works for you.

**Comment by Jillian2017** [comment]
@fab-jul Sorry for the late reply, maybe some version is not the same in my environment, I will try again. Recently I have tried to use your ms_ssim.py to train my network, but the value of ms_ssim was quite high(about 0.99) even at the very begining as I trained the network was trained from scratch. The ms_ssim' value is not reasonable, while the mean square loss is quite low. I wonder have you ever tested the ms_ssim.py before, I did not change any thing to the ms_ssim.py. Need your help, thanks a lot.

**Comment by fab-jul** [comment]
How did you try training, with which parameters? 

We have thoroughly tested `ms_ssim.py` before using it to train.

**Comment by Jillian2017** [comment]
Er, I found that in my situation the image's range value was set incorrectly,  I noticed that there was a layer of tf.clip_by_value  in your decoder network, so I adapted it in my network, and it worked.
Another problem appeared! I test an image with my trained model, and print the ms-ssim value as the way of training process, here I can get the ms-ssim value calculating in your way. After testing, I can get the reconstructed image, then I calculate the ms-ssim value with the original image using a python version [ms-ssim](https://github.com/models/blob/master/research/compression/image_encoder/msssim.py) , but these 2 values are different, e.g. the former one is 0.186 and the latter one is 0.255, so I'm very confused.
